shell/public/index.html

```diff
@@ -13,8 +13,10 @@
     <div id="app">
         <script>
             (() => {
-                const isDark = document.cookie.includes('R_THEME=auto') ?
-                    // User selected automatic theme, so use pcs (set when ui previously loaded and is either os theme or time of day based
+                // Has the user chosen to auto detect the theme.... or if they haven't chosen anything.. --> check the auto-detected theme via R_PCS
+                // Otherwise check if they've specifically selected a theme --> R_THEME
+                const isDark = document.cookie.includes('R_THEME=auto') || !document.cookie.includes('R_THEME') ?
```

Comment:
Does this mean the default theme will be dark?

Reply:
On very first load, the loading indicator will always be light 
- `R_THEME` and `R_PCS` won't exist 
- `!document.cookie.includes('R_THEME')` --> isDark = document.cookie.includes('R_PCS=dark')

Once our prefs code has ran, it will set the `R_PCS` cookie given system defaults.

Then on load...
- `R_THEME` won't exist
- `!document.cookie.includes('R_THEME')` --> isDark = document.cookie.includes('R_PCS=dark')

If the user explicitly sets `R_THEME=auto`
- document.cookie.includes('R_THEME=auto') --> isDark = document.cookie.includes('R_PCS=dark')

If the user explicitly sets `R_THEME` to anything else
- isDark = document.cookie.includes('R_THEME=dark');

So everything should work now, except really the very first load where we haven't run any code and can't determine system theme